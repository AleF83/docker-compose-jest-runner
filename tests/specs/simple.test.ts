import { execSync } from "child_process";

describe("Simple test", () => {
  test("should work", async () => {
    const result = execSync('docker ps --format "{{.Image}}: {{.State}}" | sort', { encoding: "utf8" }).trim();
    expect(result).toMatchSnapshot();
  });
});
