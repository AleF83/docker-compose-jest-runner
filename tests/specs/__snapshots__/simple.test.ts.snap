// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simple test should work 1`] = `
"mongo: running
vimagick/json-server: running"
`;
